docs/developer-guide/plugins.md

```diff
@@ -127,6 +127,26 @@ const ruleFunction = (primary, secondaryOptions) => {
 };
 ```
 
+## Quiet deprecation warnings
+
+We sometimes introduce deprecation warnings that impact plugin authors. You can individually quiet these in your plugin by handling the `emitWarning` event.
+
+For example, to quiet the deprecationg warning for `context.fix`:
+
+```js
+const original = process.emitWarning;
+process.emitWarning = function (message, options) {
+  if (
+    message &&
+    options?.type === "DeprecationWarning" &&
+    options.code === "stylelint:005"
+  )
+    return;
```

Comment:
[suggestion] This is just my opinion, but I'd recommend `return` is surrounded by `{}` for clarity:

```suggestion
  ) {
    return;
  }
```